prinz_util/src/main/scala/pl/touk/nussknacker/prinz/util/http/AbstractRestJsonClient.scala

```diff
@@ -28,13 +31,19 @@ abstract class AbstractRestJsonClient(private val baseUrl: String) {
     uri"${s"$baseUrl$relativePath"}?$params"
 
   protected def clientExceptionFromResponse(value: ResponseException[String, Exception]): RestClientException =
-    RestClientException(value.toString)
+    RestClientException(value)
 
   protected def wrapCaughtException[R](requestAction: () => R, exceptionHandler: Exception => R): R = try {
     requestAction()
   } catch {
     case e: SttpClientException => exceptionHandler(e)
   }
+
+  /*protected def defaultWrapCaughtException[RESPONSE: Manifest, F[_]]
```

Comment:
I've trying to merge that 2 'wrapping' methods but I didn't manage to deal with 'cannot resolve symbol map' error. Maybe one of u have some idea?

Reply:
oh, I see now there is no chance to do as these clients are different because one returns Future and other the Identity of request. I changed the names of these functions that you introduced to make it visible why they are different